native/chat/chat-thread-list.react.js

```diff
@@ -181,6 +199,12 @@ class ChatThreadList extends React.PureComponent<Props, State> {
           searchResults.has(item.threadInfo.id),
         );
       }
+      if (emptyItem && chatItems.length === 0) {
```

Comment:
How about:

```js
if (emptyItem && chatItems.length === 0) {
  chatItems.push({ type: 'empty', emptyItem });
}
```

This way we avoid defining `{ type: 'search', searchText }` twice